packages/validator/test/plugins/validation/2and3/schema-ibm.test.js

```diff
@@ -1,160 +1,158 @@
 const expect = require('expect');
-const yaml = require('js-yaml');
```

Comment:
My goal in this file was to remove all the tests associated with the function that I removed from schema-ibm.js, but leave all the rest.